open-edge-platform/training_extensions
# Migrate default segmentation model to OTXv2

### Summary



### How to test



```
unit/cli/utils/test_install.py::test_mim_installation
unit/cli/utils/test_install.py::test_mim_installation
  /home/yuchunli/git/otx-2x/venv/lib/python3.10/site-packages/pip/_vendor/pkg_resources/__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('zope')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
============================================================================================== 636 passed, 1 xfailed, 882 warnings in 435.27s (0:07:15) ==============================================================================================
```

### Checklist




### License

      Feel free to contact the maintainers if that's a concern.

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```
